shopfloor_mobile_base/static/wms/src/components/manual_select.js

```diff
@@ -150,12 +150,13 @@ Vue.component("manual-select", {
                 this.is_selected(rec)
             );
         },
-        selected_color_klass(modifier) {
-            return (
-                "active " +
-                this.utils.colors.color_for("item_selected") +
-                (modifier ? " " + modifier : "")
-            );
+        selected_color_klass(rec, modifier) {
+            let color = this.utils.colors.color_for("item_selected");
+            if (rec.qty_done && rec.qty_done < rec.quantity)
```

Comment:
This should not stay here. ManualSelect is a generic component.
You should pass a new prop to support custom klass handler.

Reply:
Moved as an extension of the component in shopfloor_mobile